Streetcode/Streetcode.DAL/data.csv

```diff
@@ -48,3 +48,13 @@
 ³�������;;³��������;³�������;�. ³�����;21010;���. ����� �����;49.251772;28.4606408
 ³�������;;³��������;³�������;�. ³�����;21010;���. ����������;49.2595806;28.4637077
 ³�������;;³��������;³�������;�. ³�����;21010;���. ����� �������;49.254716;28.4587164
+³�������;;³��������;³�������;�. ³�����;21010;���. ����� �����������;49.256425;28.45977
+³�������;;³��������;³�������;�. ³�����;21010;����. ����� �����������;49.2532258;28.463045
+³�������;;³��������;³�������;�. ³�����;21010;���. ���������;49.2569242;28.4551958
```

Comment:
Are you sure about commiting changes here?

Reply:
Also assume that this file in general should be added to git ignore.

Reply:
by bad, I forgot that they changing without my participation.